### PR TITLE
Add tests for browsingContext.create referenceContext parameter

### PR DESCRIPTION
**Pull request by juliandescottes**
From 
- https://phabricator.services.mozilla.com/D143652
- https://phabricator.services.mozilla.com/D143653

### CODE REVIEW

File: webdriver/tests/bidi/browsing_context/create/invalid.py [hunk 1]
```diff
@@ -4,6 +4,43 @@
...
+        url=test_page_same_origin_frame,
+        wait="complete",
+    )
+
+    all_contexts = await bidi_session.browsing_context.get_tree()
+
+    assert len(all_contexts) == 1
+    parent_info = all_contexts[0]
+    assert len(parent_info["children"]) == 1
+    child_info = parent_info["children"][0]
```
Comment (sadym-chromium) [inline webdriver/tests/bidi/browsing_context/create/invalid.py:36]: Maybe simplify, as those assertions are out of scope of the test?
```suggestion
    child_info = all_contexts[0]["children"][0]
```

**Review by sadym-chromium — APPROVED** [review]
LGTM with one optional suggestion

File: webdriver/tests/bidi/browsing_context/create/invalid.py (hunk 1 above)
Comment (whimboo) [reply webdriver/tests/bidi/browsing_context/create/invalid.py:36]: We have these assertions to make error messages easier to spot and investigate. In your case it would fail with an `IndexError`.
